pyflakes/checker.py

```diff
@@ -697,6 +697,11 @@ def ignore(self, node):
     # additional node types
     COMPREHENSION = KEYWORD = FORMATTEDVALUE = handleChildren
 
+    def ASSERT(self, node):
+        if isinstance(node.test, ast.Tuple):
```

Comment:
What about

```python
assert ()
```

Not sure why anyone would do that instead of just `assert False`, but it's always False (so has some utility), and doesn't elicit a SyntaxWarning on Python 2.7, so we probably shouldn't complain about it.